### PR TITLE
app-misc/gtypist: add patch to fix crash and bump to 2.9.5-r1

### PR DESCRIPTION
**Pull request by arjanadriaanse**
Closes: https://bugs.gentoo.org/698764
Signed-off-by: Arjan Adriaanse <arjan@adriaan.se>

### CODE REVIEW

File: app-misc/gtypist/gtypist-2.9.5-r1.ebuild [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
```
Comment (thesamesam) [inline app-misc/gtypist/gtypist-2.9.5-r1.ebuild:4]: Can we do EAPI=7 and update the rest of the ebuild appropriately?

Comment (arjanadriaanse) [reply app-misc/gtypist/gtypist-2.9.5-r1.ebuild:4]: I am not the maintainer of this package and not experienced, so my intention was to just make minimal changes that fix the bug.

Comment (thesamesam) [reply app-misc/gtypist/gtypist-2.9.5-r1.ebuild:4]: I reckon this is simple, change to EAPI=7, and possibly be able to drop eutils. See if it still works.

Comment (arjanadriaanse) [reply app-misc/gtypist/gtypist-2.9.5-r1.ebuild]: Thank you, it was indeed simple.

**Comment by thesamesam** [comment]
Looking good! Could you squash the commits? (then force push)

Thank you for making the effort to port to EAPI=7. We are trying to reduce the number of EAPI=5 in the tree especially, so this will help.

**Pull request closed without merging**